php/add.php

```diff
@@ -26,7 +26,6 @@ function die_and_log($message) {
 session_start();
 
 // Check CSRF token
-session_start();
```

Comment:
This should stay, because without it the session is not started and the token can't be checked (though I might be wrong, that's just my experience)